python/src/scipp/__init__.py

```diff
@@ -21,6 +21,7 @@
 from .table_html import to_html, make_html
 from .object_list import _repr_html_
 from ._utils import collapse, slices
+from ._utils.is_type import *
```

Comment:
Can we list by hand what we want to import? Certainly someone will add something to `is_type` in the future that we don't want public automatically?